Server/Include/ClientHandler.cpp

```diff
@@ -26,6 +27,11 @@ void ClientHandler::OnConnected(LambdaEngine::IClient* pClient)
 {
 	UNREFERENCED_VARIABLE(pClient);
 	LOG_MESSAGE("OnConnected()");
+
+	/*LambdaEngine::NetworkSegment* pPacket = pClient->GetFreePacket(69);
+	LambdaEngine::BinaryEncoder encoder(pPacket);
+	encoder.WriteString("Christoffer");
+	pClient->SendReliable(pPacket, this);*/
```

Comment:
As the general comment mentioned. If the big segments of code are not needed they should be removed, if they are needed they shouldn't be out-commented.